integration-tests/tests/network/runner.rs

```diff
@@ -861,6 +877,7 @@ pub fn ban_peer(target_peer: usize, banned_peer: usize) -> ActionFn {
               flag: Arc<AtomicBool>,
               _ctx: &mut Context<WaitOrTimeoutActor>,
               _runner| {
+            debug!(target: "network", "runner.rs ban_peer {:?}", (target_peer, banned_peer));
```

Comment:
I would recommend using the `tracing`'s built-in ability to output key-values, rather than manually formatting in the variables into the message. For example in this case I'd write

```suggestion
            debug!(target: "network", "runner.rs ban_peer", ?target_peer, ?banned_peer);
```

which will use `Debug` representation of `target_peer` and `banned_peer` separately.

Reply:
@nagisa I tried it, but it doesn't work. Please take a look at: https://github.com/near/nearcore/pull/5392

I haven't seen this syntax used in our code base. What is the name of this feature, is it a nightly feature?

Anyway, introducing new systax should be done in separate PR. I'm happy to do it. Let's try it out.

Reply:
@nagisa Oh, we are missing "message =" keyword